libs/wallet/src/connect-dialog/connect-dialog.tsx

```diff
@@ -159,21 +140,28 @@ const ConnectDialogContainer = ({
     // Immediately connect on selection if jsonRpc is selected, we can't do this
     // for rest because we need to show an authentication form
     if (connector instanceof JsonRpcConnector) {
+      connector.url = walletUrl;
```

Comment:
set url of this connector is little bit tricky, since setter of this property has some logic, which may cause losing of auth token and thus errors during connecting

Reply:
ok, I can't reproduce this connection error, so seems to be ok with this set